Wrappers/Python/cil/framework/acquisition_geometry.py

```diff
@@ -1976,112 +2224,239 @@ def set_channels(self, num_channels=1, channel_labels=None):
+
+        The rotation axis position is [0,0,0] which means that the rotation axis is assumed to be at the origin. 
+        The rotation axis direction is [0,0,1] which means that the rotation axis is assumed to be pointing in the z direction.
+
+        The default direction of the ray is [0,1,0] which means that the ray is assumed to be pointing in the y direction.
+        """
         AG = AcquisitionGeometry()
         AG.config = Configuration(units)
         AG.config.system = Parallel3D(ray_direction, detector_position, detector_direction_x, detector_direction_y, rotation_axis_position, rotation_axis_direction, units)
         return AG
 
     @staticmethod
     def create_Cone3D(source_position, detector_position, detector_direction_x=[1,0,0], detector_direction_y=[0,0,1], rotation_axis_position=[0,0,0], rotation_axis_direction=[0,0,1], units='units distance'):
-        r'''This creates the AcquisitionGeometry for a cone beam 3D tomographic system
-
-        :param source_position: A 3D vector describing the position of the source (x,y,z)
-        :type source_position: list, tuple, ndarray, optional
-        :param detector_position: A 3D vector describing the position of the centre of the detector (x,y,z)
-        :type detector_position: list, tuple, ndarray, optional
-        :param detector_direction_x: A 3D vector describing the direction of the detector_x (x,y,z)
-        :type detector_direction_x: list, tuple, ndarray
-        :param detector_direction_y: A 3D vector describing the direction of the detector_y (x,y,z)
-        :type detector_direction_y: list, tuple, ndarray
-        :param rotation_axis_position: A 3D vector describing the position of the axis of rotation (x,y,z)
-        :type rotation_axis_position: list, tuple, ndarray, optional
-        :param rotation_axis_direction: A 3D vector describing the direction of the axis of rotation (x,y,z)
-        :type rotation_axis_direction: list, tuple, ndarray, optional
-        :param units: Label the units of distance used for the configuration, these should be consistent for the geometry and panel
-        :type units: string
-        :return: returns a configured AcquisitionGeometry object
-        :rtype: AcquisitionGeometry
-        '''
+        """
+        Creates the AcquisitionGeometry for a cone beam 3D tomographic system.
+
+        After creating the AcquisitionGeometry object, the panel must be set using the `set_panel()` method and the angles must be set using the `set_angles()` method.
+        In addition,`set_channels()` can be used to set the number of channels and their labels. `set_labels()` can be used to set the order of the dimensions describing the data.
+
+        Parameters
+        ----------
+        source_position : array_like
+            A 3D vector describing the position of the source (x,y,z). With the default directions, this should lie along the y-axis.
```

Comment:
What does it mean "With the default directions, this should lie along the y-axis."?

I find it confusing as there is no default choice of this parameter and to be on the y-axis you should pass something like `(0,100,0)`? What if users don't do that?
```suggestion
            A 3D vector describing the position of the source (x,y,z).
```